server/char/list.py

```diff
@@ -1,22 +1,44 @@
-from pyraknet.bitstream import Serializable, c_uint8, c_uint16, c_uint32, c_uint64, c_bool
+"""
+Character lsit
+"""
+
+from pyraknet.bitstream import Serializable, c_int64, c_uint8, c_uint16, c_uint32, c_uint64, c_bool
+
 from plugin import Plugin, Action, Packet
-from structs import LUHeader, CString, GameVersion
+from structs import LUHeader, CString
+
 
 class CharacterList(Plugin):
+    """
+    Character list plugin
+    """
     def actions(self):
+        """
+        Returns all actions
+        """
         return [
-            Action('pkt:client_character_list_request', self.character_list_request, 10)
+            Action('pkt:character_list_request', self.character_list_request, 10)
         ]
 
     def packets(self):
+        """
+        Returns all packets
+        """
         return [
             ClientCharacterListRequest,
             CharacterListResponse
         ]
 
     def character_list_request(self, packet, address):
-        uid = self.server.connections[address]['uid']
-        characters = self.server.handle_until_value('char:characters', True, uid)
+        """
+        Handles a char list request
+        """
+        # TODO: add user to db and uncomment
```

Comment:
Per our conversation, a new user can be added via `manage.py createsuperuser`. Please revert these lines.

Reply:
already uncommented some other lines, seems like I forgot this one